### ISSUE TITLE
When will get a 2.8 compatible release? 

### ISSUE THREAD

**Issue by xCyborg** [sebbas/blender-mantaflow#32]
I mean is this project even still alive, what's going on?

**Comment by juangea** [comment]
You should follow my YouTube channel :)

You already have a 2.8.51 based build, and it will be updated when new features or bug fixes are done :)

Go to graphicall.org and you can download it there, and check this video for example (not my best, but my latest in English hehehe )

https://youtu.be/UUfRAG25No8

**Comment by semillerimages** [comment]
@juangea  that .51 build of mantaflow disables hybrid rendering.

**Comment by juangea** [comment]
No, it does not, at least on windows, I don't do the other OS builds:

![image](https://user-images.githubusercontent.com/20524718/56955109-c0e7b380-6b40-11e9-93c2-80cfe3811174.png)

If you have problems with other OS builds comment it in the graphicall comments section of that build so the person doing it is aware :)

**Comment by semillerimages** [comment]
> No, it does not, at least on windows, I don't do the other OS builds:
> 
> ![image](https://user-images.githubusercontent.com/20524718/56955109-c0e7b380-6b40-11e9-93c2-80cfe3811174.png)
> 
> If you have problems with other OS builds comment it in the graphicall comments section of that build so the person doing it is aware :)

I'm using the windows build. I've checked on three machines using cpu-z gpu-z and task manager. And yes, I know I am doing it right because in the daily builds from Blender, the hybrid rendering works fine on all of my machines.

**Comment by GottfriedHofmann** [comment]
Just compiled it on 2.8 latest - works fine
